ide/api/project.py

```diff
@@ -280,6 +283,44 @@ def begin_export(request, project_id):
 
 
 @login_required
+@require_safe
+@json_view
+def get_projects(request):
+    filters = {
+        'owner': request.user
+    }
+    exclusions = {}
+    parent_project = None
+    if request.GET.get('libraries', None):
+        filters['project_type'] = 'package'
+        parent_project = get_object_or_404(Project, pk=request.GET['libraries'], owner=request.user)
```

Comment:
You `request.GET['libraries']` enough that this could probably be assigned to a variable. Also cast to an int?